en/components/grid_toolbar.md

```diff
@@ -74,10 +78,10 @@ export class AppModule {}
 
 #### Customizing the Export
 
-When the user initiates an export process the IgxGrid's `onToolbarExporting` event is emitted. This event exposes both the export options and the exporter itself. This allows you to configure the export process.
+In addition to simply switching on and off the toolbar features, the export process can be further configured in the `onToolbarExporting` event which is emitted when the user initiates the export process. This event exposes both the exporter and the exporter's options. This allows you to configure the export process.
```

Comment:
'This allows you to configure the export process.' sentence should be removed now.